README.md

```diff
@@ -14,7 +14,7 @@ The code is reasonably clean but documentation and examples are sparse. Feel fre
 ⚠️ We welcome PRs and bug reports but making this repo a public success is not our priority. No promises on when it will be addressed!
 
 # Dependencies
-- [Unity 2020 LTS (2020.3.30+)](https://unity.com/)
+- [Unity 2021.2+](https://unity.com/)
```

Comment:
```suggestion
- [Unity 2021.3.5+](https://unity.com/)
```

This got updated due to Entities getting upped to 0.51

Reply:
I think it's still 0.5 compatible even though we upgraded. There were no changes in the library between 0.50 and 0.51 I think.